src/main/groovy/com/bmuschko/gradle/cargo/tasks/local/LocalCargoContainerTask.groovy

```diff
@@ -261,7 +261,7 @@ class LocalCargoContainerTask extends AbstractCargoContainerTask {
 
     private Map<String, String> getCargoAttributes() {
         def cargoAttributes = ['containerId': getContainerId(), 'action': getAction()]
-        if (getTimeout()) {
+        if (getTimeout() >= 0) {
```

Comment:
I believe this wouldn't cover the case where the value of the timeout property is set to null. This should probably say `getTimeout() && getTimeout >= 0`. Could you please correct?

Reply:
Thanks for clarifying. I just verified it myself: https://groovyconsole.appspot.com/script/5148289452212224. Would you mind reverting the last commit then? I will merge shortly thereafter.

Reply:
sure